project_template/webpack.config.js

```diff
@@ -14,6 +14,17 @@ const extractOtherCss = new ExtractText("styles.[hash].css");
 
 process.stderr.write(`Building with env = ${env}\n`);
 
+const MAJI_APP = /^MAJI_APP_/i;
+
+const getEnvironmentVariables = function() {
```

Comment:
I love this idea but I'm not sure it belongs in the projects webpack config.
We don't do this now with anything but perhaps it would be nice to move this functionality into the Maji library and import here.

For example something like:

```
const plugins = [
   ...require("maji/webpack").plugins,
   // app specific plugins here
]
```

This also provides an extension point for future updates where can inject new plugins into Maji apps without those apps having to manually change their webpack config. I would be careful not to inject too much magic here, but for things like this IMO it's very appropriate.

What do you think? /cc @ruisalgado @matthijsgroen @ljpengelen

Reply:
Sounds good to me!

Reply:
Sounds good to me too. It's an easy way to benefit from new Maji features by just upgrading Maji itself.